platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/hashmap/Bucket.java

```diff
@@ -24,41 +24,41 @@
 
 import com.swirlds.common.io.streams.SerializableDataOutputStream;
 import com.swirlds.common.utility.Units;
-import com.swirlds.merkledb.files.DataFileOutputStream;
 import com.swirlds.merkledb.serialize.KeySerializer;
 import com.swirlds.virtualmap.VirtualKey;
 import java.io.IOException;
 import java.io.UncheckedIOException;
+import java.nio.BufferOverflowException;
 import java.nio.ByteBuffer;
 import java.util.concurrent.atomic.AtomicInteger;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 /**
  * Class for accessing the data in a bucket. This is designed to be used from a single thread.
- * <p>
+ *
  * Each bucket has a header containing:
+ *
  * <ul>
- *     <li><b>int</b> - Bucket index in map hash index</li>
- *     <li><b>int</b> - Bucket size, total number of bytes taken by bucket including header</li>
- *     <li><b>int</b> - Number of entries in this bucket</li>
- *     <li><b>Entry[]</b>  - array of entries</li>
+ *   <li><b>int</b> - Bucket index in map hash index
```

Comment:
If this is spotless, it's wrong.  Removing end tags breaks HTML compatibility, and changes the semantics of the comments.

Reply:
I didn't change HTML tags explicitly, nor spotless complain about it. However, it could be removed by an old version of spotless, which I ran before sync with develop. Let me restore the tags.